packages/core/admin/admin/src/content-manager/components/EditViewDataManagerProvider/index.js

```diff
@@ -170,12 +203,15 @@ const EditViewDataManagerProvider = ({
     });
   }, []);
 
-  const connectRelation = useCallback(({ target: { name, value, replace } }) => {
+  /**
+   * @type {({ name: string, value: Relation, isSingleRelation: boolean}) => void}
```

Comment:
The name of the param `isSingleRelation` is out of sync with the actual function.